.github/workflows/elixir.yml

```diff
@@ -30,4 +28,10 @@ jobs:
         mix local.hex --force
         mix deps.get
     - name: Run Tests
+      env: 
+        PGUSER: 'postgres'
+        PGPASSWORD: 'postgres'
+        PGDATABASE: 'philomena_test'
+        PGPORT: 5432
+        PGHOST: db
```

Comment:
Its required for Philomena to take IP of host by looking at PGHOST.

PostgresSQL is running in different container and I cannot make it simply run it in this one.
I could forcefully install it using apt...

Reply:
@liamwhite What you think is more correct?
Make connection based on PGHOST or forcefully install PostrgresSQL to Cluster?

Reply:
You should update config/test.exs to use the right address.
https://github.com/derpibooru/philomena/blob/c79dfd5b21389d4a441c5b40da17d1eee8bdb2f9/config/test.exs#L4-L8